frontend/src/model.js

```diff
@@ -359,6 +359,42 @@ function ListUsers() {
   })
 }
 
+function CreateUser(user) {
+  var url = 'manager/create_user';
+  request('post', url, user
+  ).then(response => {
+    if(response.status === 200) {
+      console.log("user added:", response.data);
+      let returned_user = response.data;
+      setState(['users', 'index', returned_user.Username], returned_user);
+    } else {
+      console.log(response)
+    }
+  }).catch(error => {
+    if(error.response)
+      notify(`Error: ${error.response.data.message}`, 'error');
+    console.log(error.response)
+  })
+}
+
+function DeleteUser(user) {
+  var url = 'manager/delete_user';
+  request('post', url, user
+  ).then(response => {
+    if(response.status === 200) {
+      let returned_user = response.data;
+      console.log(`user ${returned_user.Username} deleted`);
+      clearState(['users', 'index', returned_user.Username], []);
```

Comment:
extraneous `, []` at the end of this call -- you can just call `clearState(path)`